tests/test_task_post.py

```diff
@@ -0,0 +1,82 @@
+from rest_framework import status
+from .common import EndpointTestCase
+
+
+class TestTaskPost(EndpointTestCase):
```

Comment:
Same topic as in #180 - how do we want to group tests?

Reply:
The main is info about what do it a test method or test class. I propose to write postfix with type GET|POST|PATCH|DELETE. For me that is readable. When do that: in classes or method or both... To discussion